modules/apps/frontend-js/frontend-js-collapse-support-web/npmscripts.config.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+module.exports = {
+	federation: {
+		mode: 'default',
```

Comment:
[This](https://github.com/liferay-frontend/liferay-portal/pull/839#discussion_r585574678) is going to kill me every time I see it. 😬

Reply:
`mode: 'weird'`

Reply:
![ommm](https://media.giphy.com/media/C8uDjPYn4c5LmVrR0I/giphy.gif)

Reply:
One day you'll get used to it.
The next day we will remove it.

Reply:
Revisiting this once more... `mode: 'weird'` works, in fact: anything that is not `default` or `disabled` is interpreted as `compatible` in `npm-scripts`.

If you find it interesting, we can use random words instead of `compatible` as a way to celebrate diversity :tada: ... `#JustSaying`